### ISSUE TITLE
[suggestion]improving the dash, lockscreen view, activator dash+another feature

### ISSUE THREAD

**Issue by alon-sht** [peterhajas/MobileNotifier#208]
In the dash there is a lot of space for every notification. Yet only a very little part of it is displayed. Same thing with the lock screen view. I think you can make more parts of the notification visible.

Also I believe you can change the notification itself somehow so we can see the whole notification.

Another feature I want to suggest is that if there are a few notifications "on hold" maybe they would "move" from one to another at the top of the screen.

**Comment by peterhajas** [comment]
If you have multiple feature requests, please break them up.

I do not understand your issues with the AlertDashboard and the lockscreen view, are they not tall enough?

**Comment by kristianpennacchia** [comment]
I agree with the first, I think it would be nice if the dash view provided more space to show more of the message. 
Ex: Mail.app gives space for five lines. Thats probably a bit too much but 3 lines would be perfect in my opinion. Enough space to see all/most of the message and decide if you want to open it.

I'm not sure how that would look for the lockscreen though, might be a bit too big.

Need more space for the double height status bar alert too. Maybe if when you tapped on the status bar it would hide the sender name/ID and show more of the message instead? Then revert when you tap on the status bar again.
